backend/fleet_management/serializers.py

```diff
@@ -16,10 +16,13 @@ class Meta:
 
 class UserSerializer(serializers.ModelSerializer):
     groups = GroupSerializer(many=True)
+    rsa_modulus_n = fields.CharField(read_only=True)
+    rsa_pub_e = fields.CharField(read_only=True)
+    rsa_priv_d = fields.CharField(read_only=True)
 
     class Meta:
         model = User
-        fields = ("id", "username", "groups")
+        fields = ("id", "username", "groups", "rsa_modulus_n", "rsa_pub_e", "rsa_priv_d")
```

Comment:
Do we need to pass the private key?

Reply:
Yes, this one is a different serializer, it's used only for getting the details of currently logged in user (via `/api/users/me`). And we want these details but only for the _logged in_ user.
Passenger serializer defined below (used by Drivers) does not expose the private key.